test/make_dist.py

```diff
@@ -131,12 +131,14 @@ def download_dist(wait=False):
         tar_path = os.path.realpath(names[0])
 
     # Extract node_modules and dist locally for speeding up the build and allowing integration tests to run
-    unpack_dirs = [d for d in ["dist", "node_modules", "package-lock.json"] if not os.path.exists(d)]
+    unpack_dirs = [d for d in ["dist", "node_modules", "package-lock.json", "tools/debian/copyright"] if not os.path.exists(d)]
     if unpack_dirs:
         message("make_dist: Extracting from tarball:", ' '.join(unpack_dirs))
         prefix = os.path.basename(tar_path).split('.tar')[0] + '/'
         prefixed_unpack_dirs = [prefix + d for d in unpack_dirs]
         subprocess.check_call(["tar", "--touch", "--strip-components=1", "-xf", tar_path] + prefixed_unpack_dirs)
+        # after the above, dist manifests may be newer than copyright due to tar touching order, avoid rebuild
```

Comment:
It does not even do it in alphabetical order? Not need to answer, that is a weird one though :P

Reply:
I strongly suspect this depends on the order of files in the tar -- this is a stream processor by nature, so I guess it iterates over the files in the tar, filters according to the command line, and writes them out in that order.